NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -352,6 +351,7 @@ private List<Entity> ConvertComponentPrefabsToEntities(List<PrefabAsset> prefabs
                 Entity prefabEntity = new Entity(transform.LocalPosition,
                              transform.LocalRotation,
                              transform.LocalScale,
+
```

Comment:
A new line? :D

Reply:
Honestly, I don't even try anymore because autoformatting is annoying to setup in a way to Nitrox has its code style (and also some code style don't seem to work properly for me).